charts/shoot-core/charts/csi-alicloud/templates/csi-attacher-dep.yaml

```diff
@@ -1,4 +1,68 @@
 {{- if .Values.enabled }}
+{{- if semverCompare ">= 1.13" .Values.kubernetesVersion -}}
```

Comment:
Why do we need this `if`-`else` block around the whole deployment?!

Reply:
For easy maintainance.
-----
Actually, we plan only support k8s v1.13.+ because
1. CSI is more stable.
2. CSI Alicloud plugin v1.0 is not compatible with v0.3.0, because they changed the plugin name.
 We  need 2 release cycle to retire v1.11.+.
 The PR is the first cycle and supports v1.11(Only for delete) & v1.13.
 In the next release cycle, we will remove v1.11 part -- just remove else part

Reply:
ok